packages/starknet-snap/src/utils/constants.ts

```diff
@@ -11,6 +11,8 @@ export const MAXIMUM_NETWORK_NAME_LENGTH = 64;
 export const MAXIMUM_TOKEN_NAME_LENGTH = 64;
 export const MAXIMUM_TOKEN_SYMBOL_LENGTH = 16;
 
+export const TRANSFER_SELECTOR_HEX = '0x83afd3f4caedc6eebf44246fe54e38c95e3179a5ec9ea81740eca5b482d12e';
```

Comment:
@jonesho just curious how is this derived?

Reply:
@onetrickwolf its from the starknet utils which would translate the function name "transfer" to hex string. FYI https://util.turbofish.co/